### DISCUSSION TITLE
iCloud directory on macos

### DISCUSSION THREAD

**Discussion by niklod** [gokcehan/lf#855] [Q&A] [answered]
Hey guys, how i can add iCloud drive directory to lf? Didn't find any info about it

**Comment by robertranjan** [comment, marked as answer]
It's an indirect answer to your question. I do not know how to add it to `lf`.

here is my iCloud drive: `~/Library/Mobile\ Documents/com~apple~CloudDocs`

You may try create a symlink to your home as below
```
cd ${HOME}
ln -sfn ${HOME}/Library/Mobile\ Documents/com~apple~CloudDocs iCloud
```
then open `lf` and navigate to ${HOME}

**Reply by CatsDeservePets** [reply]
@robertranjan is correct.

I have this in my personal config:
```sh
map gi cd '~/Library/Mobile Documents/com~apple~CloudDocs'
```
